### PR TITLE
fix(#753): add autoComplete attribute for remove suggestion of passwo…

### PR DESCRIPTION
**Pull request by gfr2003**
Add autoComplete attribute for remove suggestions of password managers

### CODE REVIEW

**Comment by gfr2003** [comment]
Hello, @FelixMalfait, it seems that it broke in the tests but it is not related to the issue... What do you think?

File: front/src/modules/ui/table/editable-cell/types/EditableChip.tsx [hunk 1]
```diff
@@ -67,6 +67,7 @@ export function EditableCellChip({
         <StyledInplaceInput
           placeholder={placeholder || ''}
           autoFocus
+          autoComplete="off"
```
Comment (charlesBochet) [inline front/src/modules/ui/table/editable-cell/types/EditableChip.tsx:70]: Thank you!
you would also need to make this modification in other types of @/ui/table/editable-cell and @/ui/editable-fields

**Comment by charlesBochet** [comment]
> Hello, @charlesBochet and @FelixMalfait, it seems that it broke in the tests but it is not related to the issue... What do you think?

Yes, I have re-run the tests, it should be good!

**Review by magrinj — APPROVED** [review]

File: front/src/modules/ui/inplace-input/components/InplaceInputDoubleText.tsx [hunk 2]
```diff
@@ -31,6 +31,8 @@ export function InplaceInputDoubleText({
 }: OwnProps) {
   return (
     <StyledContainer>
+      <InplaceInputTextEditMode
```
Comment (charlesBochet) [inline front/src/modules/ui/inplace-input/components/InplaceInputDoubleText.tsx:34]:
```suggestion
```

File: front/src/modules/ui/inplace-input/components/InplaceInputDoubleText.tsx [hunk 3]
```diff
@@ -31,6 +31,8 @@ export function InplaceInputDoubleText({
...
+        autoComplete="off"
```
Comment (charlesBochet) [inline front/src/modules/ui/inplace-input/components/InplaceInputDoubleText.tsx:35]:
```suggestion
```

File: front/src/modules/ui/inplace-input/components/InplaceInputDoubleText.tsx [hunk 4]
```diff
@@ -39,7 +41,10 @@ export function InplaceInputDoubleText({
           onChange(event.target.value, secondValue);
         }}
       />
+      <InplaceInputTextEditMode
```
Comment (charlesBochet) [inline front/src/modules/ui/inplace-input/components/InplaceInputDoubleText.tsx:44]:
```suggestion
```

File: front/src/modules/ui/inplace-input/components/InplaceInputDoubleText.tsx [hunk 5]
```diff
@@ -39,7 +41,10 @@ export function InplaceInputDoubleText({
...
+        autoComplete="off"
```
Comment (charlesBochet) [inline front/src/modules/ui/inplace-input/components/InplaceInputDoubleText.tsx:45]:
```suggestion
```

**Review by charlesBochet — APPROVED** [review]

**Comment by charlesBochet** [comment]
Merging it, thank you!

**Pull request merged**